docs/resources/Channel.md

```diff
@@ -1174,14 +1176,16 @@ Create a new [invite](#DOCS_RESOURCES_INVITE/invite-object) object for the chann
 
 | Field                 | Type      | Description                                                                                                                               | Default          |
 | --------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ---------------- |
-| max_age               | integer   | duration of invite in seconds before expiry, or 0 for never. between 0 and 604800 (7 days)                                                | 86400 (24 hours) |
+| max_age\*             | integer   | duration of invite in seconds before expiry, or 0 for never. between 0 and 2592000 (30 days)                                              | 86400 (24 hours) |
 | max_uses              | integer   | max number of uses or 0 for unlimited. between 0 and 100                                                                                  | 0                |
 | temporary             | boolean   | whether this invite only grants temporary membership                                                                                      | false            |
 | unique                | boolean   | if true, don't try to reuse a similar invite (useful for creating many unique one time use invites)                                       | false            |
 | target_type           | integer   | the [type of target](#DOCS_RESOURCES_INVITE/invite-object-invite-target-types) for this voice channel invite                              |                  |
 | target_user_id        | snowflake | the id of the user whose stream to display for this invite, required if `target_type` is 1, the user must be streaming in the channel     |                  |
 | target_application_id | snowflake | the id of the embedded application to open for this invite, required if `target_type` is 2, the application must have the `EMBEDDED` flag |                  |
 
+\* If the channel is within a non-community server, the value of `max_age` cannot be 0—it must be between 1 and 2592000.
```

Comment:
"0—it" couldn't / shouldn't that be "0, it"

A bit confused with the wording, might be my german tho :p

Reply:
It is valid here but could be spaced out from the other words for clarity

```suggestion
\* If the channel is within a non-community server, the value of `max_age` cannot be 0 — it must be between 1 and 2592000.
```

Reply:
This confusion, if any, can be averted by simply removing the superfluous wording.

```suggestion
\* If the channel is within a non-community server, the value of `max_age` must be between 1 and 2592000.
```